pkg/controller/v1beta1/inferenceservice/components/component.go

```diff
@@ -49,7 +49,7 @@ func addStorageSpecAnnotations(storageSpec *v1beta1.StorageSpec, annotations map
 	}
 	if storageSpec.Path != nil {
 		annotations[constants.StorageInitializerSourceUriInternalAnnotationKey] =
-			fmt.Sprintf("%s://<bucket-placeholder>/%s", credentials.UriSchemePlaceholder,
```

Comment:
The reason we have `<bucket-placeholder>` here is because the storageSpec.Path is empty.

Reply:
Can if add an if statement here for if the `credentials.UriSchemePlaceholder == "hdfs"`, then assign it without `<bucket-placeholder>`?

Reply:
`credentials.UriSchemePlaceholder` only ever contains `<scheme-placeholder>`, not the actual scheme being used